algorithms/ceres-error-terms/include/ceres-error-terms/visual-error-term-inl.h

```diff
@@ -183,7 +183,7 @@ bool VisualReprojectionError<CameraType, DistortionType>::Evaluate(
 
     // These Jacobians will be used in all the cases.
     J_p_C_fi_wrt_p_C_I = Eigen::Matrix3d::Identity();
-    J_p_C_fi_wrt_q_C_I = common::skew(p_C_fi);
+    J_p_C_fi_wrt_q_C_I = common::skew(R_C_I * p_I_fi);
```

Comment:
quick question for clarification: does it make a difference if you do `R_C_I * common::skew(p_I_fi);`? 
I'm asking because in line `222` we have almost the exact same jacobian: `J_p_C_fi_wrt_q_I_M = R_C_I * common::skew(p_I_fi);` but wrt. to `q_I_M`

Reply:
Yep, it's different. For `R_C_I * common::skew(p_I_fi);` you have
```
|r11 r12 r13|   |  0 -p3  p2|   |r12*p3-r13*p2 r13*p1-r11*p3 r11*p2-r12*p1|
|r21 r22 r23| x | p3   0 -p1| = |r22*p3-r23*p2 r23*p1-r21*p3 r21*p2-r22*p1|
|r31 r32 r33|   |-p2  p1   0|   |r32*p3-r33*p2 r23*p1-r31*p3 r31*p2-r32*p1|
```
while `R_C_I * common::skew(p_I_fi);` is
```
      |r11 r12 r13|   |p1|          |r11*p1+r12*p2+r13*p3|        |a|   | 0 -c  b|
skew( |r21 r22 r23| x |p2| ) = skew |r21*p1+r22*p2+r23*p3| = skew |b| = | c  0 -a|
      |r31 r32 r33|   |p3|          |r31*p1+r32*p2+r33*p3|        |c|   |-b  a  0|
```